lib/individual/info_page.dart

```diff
@@ -1,309 +1,361 @@
+                                fontWeight: FontWeight.w700),
+                          ),
+                          SizedBox(
+                            height: MediaQuery.of(context).size.height * 0.104,
+                            width: MediaQuery.of(context).size.height * 0.43,
+                            child: TextField(
+                              decoration: InputDecoration(
+                                  fillColor: Color.fromRGBO(199, 226, 228, 1),
+                                  filled: true,
+                                  hintText: 'About\n',
+                                  border: OutlineInputBorder(
+                                      borderRadius: BorderRadius.circular(
+                                          MediaQuery.of(context).size.height *
+                                              0.02))),
+                            ),
+                          ),
+                          SizedBox(
+                            height: MediaQuery.of(context).size.height * 0.02,
+                          ),
+                          SizedBox(
+                            width: MediaQuery.of(context).size.height * 0.43,
+                            height: MediaQuery.of(context).size.height * 0.06,
+                            child: ElevatedButton(
+                              onPressed: () {
+                                Navigator.of(context)
+                                    .pushNamed(MyDonationsScreen.routeName);
+                              },
+                              style: ElevatedButton.styleFrom(
+                                primary: Color.fromRGBO(199, 226, 228, 1),
+                                shape: RoundedRectangleBorder(
+                                    borderRadius: BorderRadius.circular(
+                                        MediaQuery.of(context).size.height *
+                                            0.0076)),
+                              ),
+                              child: Text(
+                                'My Donations',
+                                style: TextStyle(
+                                    fontSize:
+                                        MediaQuery.of(context).size.height *
+                                            0.028,
+                                    fontWeight: FontWeight.w700),
+                              ),
+                            ),
+                          )
+                        ],
+                      ),
+                    ),
+                  ))
+            ],
+          ),
+        ),
+        bottomNavigationBar: CustomNavigationBar(currentIndex: 3));
+  }
+}
+
+class individualdrawer extends StatefulWidget {
+  const individualdrawer({Key? key}) : super(key: key);
+
+  @override
+  State<individualdrawer> createState() => _individualdrawerState();
+}
+
+class _individualdrawerState extends State<individualdrawer> {

```

Comment:
Since there is no state to be changed in the Drawer, I believe it would be better to let the drawer be a stateless widget. Please change appropriately

Reply:
Resolved in commit 9fa1a25f556674d49c11c5905c23eedf68cd224d :+1: